bot/exts/info/source.py

```diff
@@ -19,18 +19,30 @@ class BotSource(commands.Cog):
     def __init__(self, bot: Bot):
         self.bot = bot
 
-    @commands.command(name="source", aliases=("src",))
-    async def source_command(self, ctx: commands.Context, *, source_item: SourceConverter = None) -> None:
+    @app_commands.command(name="source")
+    async def source_command(
+        self,
+        interaction: Interaction,
+        *,
+        source_item: app_commands.Transform[SourceType, SourceTrasnformer] = None
+    ) -> None:
         """Display information and a GitHub link to the source code of a command, tag, or cog."""
         if not source_item:
             embed = Embed(title="Bot's GitHub Repository")
             embed.add_field(name="Repository", value=f"[Go to GitHub]({URLs.github_bot_repo})")
             embed.set_thumbnail(url="https://avatars1.githubusercontent.com/u/9919")
-            await ctx.send(embed=embed)
+            await interaction.response.send_message(embed=embed)
             return
 
-        embed = await self.build_embed(source_item)
-        await ctx.send(embed=embed)
+        embed = None
+        ephemeral = False
+        if isinstance(source_item, str):
+            description = f"**Unable to convert '{source_item}' to valid command, tag, or Cog.**"
```

Comment:
Maybe staying consistent with capitalization is better here

Reply:
Updated in https://github.com/python-discord/bot/pull/2446/commits/525fb54aed18526ed54e995e6e5bab6721c49baf